testing/test-setup/src/lib/test-folder.setup.ts

```diff
@@ -10,5 +12,20 @@ export async function cleanTestFolder(dirName: string) {
 }
 
 export async function teardownTestFolder(dirName: string) {
-  await rm(dirName, { recursive: true, force: true });
+  try {
+    await rm(dirName, {
+      recursive: true,
+      force: true,
+      // eslint-disable-next-line no-magic-numbers
+      maxRetries: 3,
+      retryDelay: 100,
```

Comment:
@BioPhoton I also made the retry delay smaller and I would even say, lets have just two of `maxRetries` because if it is the filesystem lock problem on windows, then nothing will help and we just unnecessarily wait and if it is something else 2nd retry should pass.

Reply:
Thx! also implemented it! It's a good improvement.